### PR TITLE
[addon velaux] add event-view

### LINKED ISSUE #576: [Addon velaux] add view of `collect-events`
Right now we can only collect events for pods, we want to collect events for other Kind such as `Deployment` too.

### PR DESCRIPTION
**Pull request by suwliang3**
### Description of your changes

Fixes #576 

### How has this code been tested?
curl http://localhost:8000/api/v1/query?velaql=event-view{name=xxxx,namespace=xxxxx,cluster=xxxx,type=xxxx}

### Checklist

I have:

- [ ] Title of the PR starts with type (e.g. `[Addon]` , `[example]` or `[Doc]`).
- [ ] Updated/Added any relevant [documentation](https://kubevela.io/docs/reference/addons/overview) and [examples](https://github.com/kubevela/catalog/tree/master/examples).
- [ ] New addon should be put in [experimental](https://github.com/kubevela/catalog/tree/master/experimental/addons).
- [ ] Update addon should modify the `version` in `metadata.yaml` to generate a new version.

####  Verified Addon promotion rules

If this pr wants to promote an experimental addon to verified, you must check whether meet these conditions too:
  - [ ] This addon must be tested by addon's [e2e-test](./test/e2e-test/addon-test) to guarantee this addon can be enabled successfully.
  - This addon must have some basic but necessary information.
    - [ ] An accessible icon url and source url defined in addon's `metadata.yaml`.
    - [ ] A detail introduction include a basic example about how to use and what's the benefit of this addon in `README.md`.
    - [ ] Also provide an introduction in KubeVela [documentation](https://kubevela.net/docs/reference/addons/overview).
    - [ ] It's more likely to be accepted if useful examples provided in example [dir](examples/).

### CODE REVIEW

File: addons/velaux/views/event-view.cue [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+		kind:       schema[parameter.type].kind
+		metadata: {
+			name:      parameter.name
+			namespace: parameter.namespace
+		}
+	}
+	cluster: parameter.cluster
+}
+
+eventList: ql.#SearchEvents & {
```
Comment (wonderflow) [inline addons/velaux/views/event-view.cue:39]: Do we really have `SearchEvents`?

Comment (suwliang3) [reply addons/velaux/views/event-view.cue:39]: yes

**Comment by wangyikewxgm** [comment]
please upgrade the addon version.

**Comment by suwliang3** [comment]
@barnettZQG  Review please.

**Review by wonderflow — DISMISSED** [review]
LGTM, please fix the conflict

File: addons/velaux/views/event-view.cue [hunk 2]
```diff
@@ -0,0 +1,55 @@
+import (
+	"vela/ql"
+)
+
+parameter: {
+	name:      string
+	namespace: string
+	cluster:   *"" | string
+	type:      string
```
Comment (barnettZQG) [inline addons/velaux/views/event-view.cue:9]:
```suggestion
	type:      "deployment" | "statefulset" | "pod"
```

File: addons/velaux/views/event-view.cue [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+		apiVersion: schema[parameter.type].apiVersion
+		kind:       schema[parameter.type].kind
+		metadata: pod.value.metadata
+	}
+	cluster: parameter.cluster
+}
+
+status: {
+	if eventList.err == _|_ {
+			events: eventList.list
```
Comment (barnettZQG) [inline addons/velaux/views/event-view.cue:50]: Format this line.

**Review by barnettZQG — APPROVED** [review]

**Pull request merged**